src/java.base/share/classes/java/security/InvalidParameterException.java

```diff
@@ -58,4 +58,44 @@ public InvalidParameterException() {
     public InvalidParameterException(String msg) {
         super(msg);
     }
+
+    /**
+     * Constructs an {@code InvalidParameterException} with the
+     * specified detail message and cause. A detail message is a {@code String} that describes
```

Comment:
Keep these lines within 80 character width to be consistent with the rest of the file.